src/main/webapp/assets/js/feedScript.js

```diff
@@ -644,28 +658,47 @@ async function addPosts(posts) {
     postCard.setAttribute('id', post.id);
     postCard.setAttribute('tabindex', '0');
 
+    // Adds the image from the image serving URL.
+    // If there is not a valid URL, then display the stock SVG.
+    const cardImage = document.createElement('img');
+    cardImage.setAttribute('class', 'card-image');
+    let imageSource = post.imageServingUrl;
+    if (imageSource === 'no image') {
+      imageSource = './assets/svg/forkandknife.svg';
+    }
+    cardImage.setAttribute('src', imageSource);
+    // Catches unexpected errors with the image source.
+    cardImage.onerror = function() {
+      cardImage.setAttribute('src', './assets/svg/forkandknife.svg');
+    };
```

Comment:
smart! does this occur if you try to fetch a link that isn't actually an image?

Reply:
Yes it does!